gems/pending/openstack/openstack_handle/handled_list.rb

```diff
@@ -23,6 +23,18 @@ def handled_list(collection_type, options = {})
                 "in provider: #{@os_handle.address}. Message=#{err.message}"
       _log.warn err.backtrace.join("\n")
       []
+    rescue StandardError => err
```

Comment:
No need to explicitly capture StandardError as that's the default...you can just do:  `rescue => err`